src/main/java/org/apache/mesos/offer/TaskUtils.java

```diff
@@ -18,6 +21,8 @@
  */
 public class TaskUtils {
     private static final Logger LOGGER = LoggerFactory.getLogger(TaskUtils.class);
+    private static final int CONFIG_TEMPLATE_LIMIT_BYTES = 100 * 1024; // 100KB
```

Comment:
This number is an arbitrary limit just to reduce the likelihood of TaskInfo growing beyond some reasonable level. Bumped to 512KB.